internal/actioninfo/actioninfo.go

```diff
@@ -1,9 +1,27 @@
 package actioninfo
 
+import (
+	"fmt"
+	"log"
+)
+
 type DataParser interface {
-	// TODO: добавить методы
+	Parse(string) error
+	ActionInfo() (string, error)
 }
 
 func Info(dataset []string, dp DataParser) {
-	// TODO: реализовать функцию
+	for _, v := range dataset {
+		err := dp.Parse(v)
+		if err != nil {
+			log.Println(err)
+			continue
+		}
+		x, err := dp.ActionInfo()
```

Comment:
Лучше более информативно называть переменные
```suggestion
		info, err := dp.ActionInfo()
```